### ISSUE TITLE
Create space for protected content in Private Cloud WordPress

### ISSUE THREAD

**Issue by ty2k** [BCDevOps/developer-experience#2698]
Based on the findings in #2262, create a space in the Private Cloud WordPress site for protected content available to IDIR users only.

**Additional context**
- #2262

**Definition of done**
- [x] Create the space: https://cloud.gov.bc.ca/private-cloud/protected-documents/
- [x] Train Emma on how to create content there
- [x] Add a link to the protected documents from the secondary header section of the page (along with the links to "Our service", "Our products and tools", "Your team", "Support and community", and "About us"

**Comment by ty2k** [comment]
Using the Documize audit spreadsheet, I've copied over the **text** of the pages that will be moving from Documize to WordPress. Note that I haven't copied images or PDFs over, and these may or may not be able to live in WordPress in a protected fashion - this will require more research effort beyond the scope of this ticket.

I have instructed Emma via Rocket.Chat on where the protected content resides, but as we have migrated our data to our production instance without having DNS/routing in place for her to access the site, I am marking this as blocked by #2544.

**Comment by ty2k** [comment]
In our Sprint Review meeting on August 3, Olena made the decision that images that go into the protected pages in WordPress will be designed to be suitable for public consumption. It is acceptable that these images can be accessed by members of the public despite appearing in "protected" documents. Steven was present and Olena indicated that he should design his platform architecture image with that requirement in mind. Shelly will perform her own content audit of the items in Documize to ensure that nothing else has to be brought into WordPress.

**Comment by ty2k** [comment]
I've added a public page called [Protected Documents](https://cloud.gov.bc.ca/private-cloud/protected-documents/) that can be viewed by any user or search engine, regardless of authentication. That page includes instructions asking IDIR users to log in using the "Login with Keycloak" button in the site footer:

<img width="1840" alt="Protected documents page for an unauthenticated user" src="https://user-images.githubusercontent.com/25143706/182928733-e1c128d2-457c-467c-bd0a-f96de0ff736a.png">

When a user signs in with their IDIR for the first time, they are assigned the custom role "Subscriber with Read Private", which gives them the standard `read` access plus `read_private_posts` and `read_private_pages` permissions. Once they return to the Protected Documents page, they will see a link styled as a button to take them to the Protected Documents index page where links to individual protected docs live:

<img width="1840" alt="Protected Documents for an authenticated user" src="https://user-images.githubusercontent.com/25143706/182929259-b205e440-8a4a-4802-8168-09b94b35ad65.png">

The private Protected Documents Index page will 404 if an unauthenticated user or search engine tries to access it. Authenticated users will see the list of protected documents:

<img width="1840" alt="Protected Documents Index page view for an authenticated user" src="https://user-images.githubusercontent.com/25143706/182929559-004b3a1c-61a8-4f4e-8bc3-ca81e2090366.png">

I added a link to the main menu navigation so that users can find the introductory Protected Documents page. In my opinion, it might make more sense to have this link in the footer next to the Login with Keycloak button, since this feature is likely going to be used by only a small portion of the site's visitors. I've reached out to our UX designer Alex Lloyd to get her feelings on this, and I can change the location of this button as needed outside the scope of this ticket.
